### DISCUSSION TITLE
Release v0.21.1 Beta

### DISCUSSION THREAD

**Discussion by PrestonN** [FreeTubeApp/FreeTube#5389] [Announcements]
## Summary

This release fixes the recent major issue of videos unable to play when using the Local API.

## Fixes
* Update YouTube.js which fixes the recent video playback issue with the local API by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/5384
* Fixed an issue where the scroll down arrow in the live chat window would persist even if you scrolled to the bottom by @kommunarr in https://github.com/FreeTubeApp/FreeTube/pull/5302
* Fixed an issue where navigating away from an age-restricted video would cause the app to lock out by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/5343
* Fixed an issue where toast messages during Invidious API errors would not show a proper error message by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/5380
* Fixed an issue where generating DASH manifest files were not being done properly when using the Invidious API by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/5387

## New Contributors
* @Jaishree2310 made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/5342
* @4yman-0 made their first contribution in https://github.com/FreeTubeApp/FreeTube/pull/5348

**Full Changelog**: https://github.com/FreeTubeApp/FreeTube/compare/v0.21.0-beta...v0.21.1-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.21.1-beta'>Release v0.21.1 Beta</a>.</em>

**Comment by ACR-Jeff** [comment]
You all rock, Appreciate everything you all do. Thank you

**Comment by muelsyse-az** [comment]
With this release, at least on the Flatpak version, forcing the app to run on Wayland has not shown any significant issues at all (all versions below this one had an issue where full-screening a video caused a crash #5304, which was the main blocker for me).

Forcing the app to run on Wayland-only mode requires you to set an environment variable `ELECTRON_OZONE_PLATFORM_HINT=auto`, disable the X11 socket and enable the Wayland socket. The environmental variable is necessary as without it the application will simply not launch on startup because Freetube/Electron (I have no idea which are the culprit) expects an X11 session by default.

It would be nice if more people would test out Freetube Wayland-only, because it seems that running Freetube using Wayland by default (with a Fallback to X11) is an interesting prospect.

I'm currently running Fedora Kinoite 40 fyi.

**Reply by absidue** [reply]
> The environmental variable is necessary as without it the application will simply not launch on startup because Freetube/Electron (I have no idea which are the culprit) expects an X11 session by default.

Electron defaults to X11 as Wayland support in Chromium is still considered experimental.
